### DISCUSSION TITLE
Welcome to awesome-3d-tiles Discussions!

### DISCUSSION THREAD

**Discussion by pka** [pka/awesome-3d-tiles#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by renzpoogi** [comment]
Hi! Is it possible to tile .b3dm file to 3d tile ? Thanks!

**Reply by mittermichal** [reply]
you can extract gltf from it with https://github.com/CesiumGS/3d-tiles-tools#convertb3dmtoglb and then tile the glTF in Cesium Ion

**Reply by renzpoogi** [reply]
Thanks for the input @mittermichal !

**Comment by pedroermarinho** [comment]
What are the main alternatives for hosting 3D Tiles?

**Reply by mittermichal** [reply]
can be any http server that can serve static files, e.g. on my machine I just run `npx http-server --cors` inside folder where I have 3D Tiles datasets
CORS headers are needed if you want to load 3D Tiles in browser on different domain than source like https://sandcastle.cesium.com/

**Reply by pedroermarinho** [reply]
Thank you for your response. Do you know of anything similar to GeoServer for 3D Tiles, or if it's possible to store 3D Tiles in Azure Blob Storage?

I already have GeoServer integrated with Azure Blob Storage to handle GeoTIFFs on my server, but I haven't found support for 3D Tiles.

**Reply by mittermichal** [reply]
I don't, chatGPT says that all you need to do to serve folder from Azure Blob Storage is to create container, upload folder and set the container to be public
